transcom/mymove
# Update flipt to same as deployed version: v1.26.1

## Summary

This updates the local development configuration to use the same version of flipt as what is deployed.


## Verification Steps for Reviewers

### How to test

1. `./scripts/server-dev --flipt --standalone`
1. Visit <http://adminlocal:8080/system/feature-flags>
1. Use `Boolean` for the Type and `boolean_flag` for the Key
1. See that it evaluates correctly
